install/README.md

```diff
@@ -13,14 +13,16 @@ The [updateVersion.sh](updateVersion.sh) script is used to update image versions
 * `-p <hub>,<tag>` new pilot image
 * `-x <hub>,<tag>` new mixer image
 * `-c <hub>,<tag>` new ca image
-* `-r <tag>` new proxy tag
+* `-a <hub>,<tag>` specifies same hub and tag for pilot, mixer, proxy, and istio-ca containers
```

Comment:
Could use use -hub foo -tag bar ? Instead of the comma separate thing, which is pretty weird.

Reply:
BTW - separate PR is fine, just cosmetic.

Reply:
The comma is unusual though I'm just the messenger.  I copied over what's in the usage() for updateVersion.sh and git says it's been that way for at least 7 months.  updateVersion.sh is going away so it's probably not a productive use of time to change this use of commas.